src/inmanta/server/services/paramservice.py

```diff
@@ -282,7 +283,7 @@ async def list_params(self, env: data.Environment, query: Dict[str, str]) -> Api
                 "parameters": params,
                 "expire": self._fact_expire,
                 # Return datetime in UTC without explicit timezone offset
-                "now": util.datetime_utc_isoformat(datetime.datetime.now()),
+                "now": util.datetime_iso_format(datetime.datetime.now(), in_local_tz=self._server_tz_aware_timestamps),
```

Comment:
I wonder why we convert it here in the first place. I would think that the json encoder employed by `RESTHandler` would take care of it. Could you see what happens when you use a plain `datetime` object here rather than a string?

Reply:
Changed to a plain datetime object